### ISSUE TITLE
Remove all code related to updating NsiqCppStyle from a remote repository

### ISSUE THREAD

**Issue by mosherubin** [kunaltyagi/nsiqcppstyle#42]
### The Problem

`NsiqCppStyle` has anachronistic code related to updating the local NsiqCppStyle code base.  Apparently, NsiqCppStyle was meant to check the code repo at `http://nsiqcppstyle.appspot.com`.  Now that NsiqCppStyle is hosted on GitHub, this "feature" is no longer relevant and should be removed.

### Technical Details

Vestiges of remotely updating NsiqCppStyle can be found in three source files:

- nsiqcppstyle_exe.py
- agent.py
- minjson.py

`nsiqcppstyle_exe.main()` is the starting point:

```
...
import updateagent.agent
...
def main(argv=None):
    ...
        updateNsiqCppStyle = False
        for o, a in opts:
            ...
            elif o == "--update":
                updateNsiqCppStyle = True
            elif o == "--no-update":
                updateNsiqCppStyle = False
            ...

        if updateNsiqCppStyle:
            console.Out.Ci(console.Separator)
            try:
                updateagent.agent.Update(version)
            except Exception as e:
                console.Out.Error(e)
        ...
```

The above code makes a call to `updateagent.agent.Update(currentVersion)`, which does the heavy work.  `Update()` also imports `updateagent.minjson`, which is used to read in JSON data.  Now that JSON is fully supported in Python 3, `minjson.py` is no longer relevant.

In short, the entire `updateagent` folder should be removed, and related changes made to the remaining code base.

**Comment by mosherubin** [comment]
The `Version` class in `updateagent/agent.py` might be a useful generic class for analyzing version strings, unrelated to the apparently deprecated update mechanism in NsiqCppStyle.  Should this be the case (it might not, in which case ignore this comment), then:

- We should move the class to the `nsiqcppstyle_util.py` file
- Consider keeping the `Version` unit/regression test code found in `nsiqunittest/nsiqcppstyle_update_unittest.py`

**Comment by kunaltyagi** [comment]
I don't think there's much left now.

Also, I don't think we need the version unit tests. Python has good support for version testing (with a dependency, and it's almost a required one to publish: https://github.com/pypa/packaging)

Since I want to publish this to pypi to make update easy, and I deleted that code wholesale, we can ignore that concern.

**Comment by mosherubin** [comment]
Before I close the issue, it should be clear that in MR "[Add argparse for better readability #53](https://github.com/kunaltyagi/nsiqcppstyle/pull/53)" you removed all vestiges of the original NsiqCppStyle updating mechanism.
